docs/user-guide/uploading.ipynb

```diff
@@ -16,8 +16,7 @@
     "client = Client.from_token(url=\"https://scicat.ess.eu/api/v3\",\n",
     "                           token=...,\n",
     "                           file_transfer=SSHFileTransfer(\n",
-    "                               host=\"login.esss.dk\",\n",
-    "                               remote_base_path=\"/somewhere/on/remote/\"\n",
+    "                               host=\"login.esss.dk\"\n",
```

Comment:
Remove the following line right under this block of code from the notebook:
`This code is identical to the one used for [downloading](https://github.com/SciCatProject/scitacean/blob/c5fd4c786c8392a6c90ef547955aa0fbdcf6f54d/docs/user-guide/downloading.ipynb) except for the remote_base_path which tells SSHFileTransfer where to put our files.`

Reply:
I already change the wording that I mentioned in my comment.